src/python/grpcio_tests/tests_aio/unit/channel_test.py

```diff
@@ -25,6 +24,167 @@
 from tests_aio.unit._test_base import AioTestBase
 
 
+class TestAioRpcError(unittest.TestCase):
+    _TEST_INITIAL_METADATA = ("initial metadata",)
+    _TEST_TRAILING_METADATA = ("trailing metadata",)
+
+    def test_attributes(self):
+        aio_rpc_error = aio.AioRpcError(
+            grpc.StatusCode.CANCELLED,
+            "details",
+            initial_metadata=self._TEST_INITIAL_METADATA,
+            trailing_metadata=self._TEST_TRAILING_METADATA)
+        self.assertEqual(aio_rpc_error.code(), grpc.StatusCode.CANCELLED)
+        self.assertEqual(aio_rpc_error.details(), "details")
+        self.assertEqual(aio_rpc_error.initial_metadata(),
+                         self._TEST_INITIAL_METADATA)
+        self.assertEqual(aio_rpc_error.trailing_metadata(),
+                         self._TEST_TRAILING_METADATA)
+
+
+class TestCall(AioTestBase):
```

Comment:
Should we split this into a separate file? Unit test files grows fast, let's keep one test case class per file from start.

Reply:
the same comment that I made for the `AioRpcError`, wanted to have coherence between modules and unit test files. My suggestion would be, let's move the logic under the `_call.py` file and having a new test file called `call_test.py`

Reply:
That makes sense. Can we create a TODO issue, and add a comment here?

Reply:
done in this PR